run-tests.php

```diff
@@ -586,7 +586,9 @@ function main(): void
                     }
                     break;
                 case '--show-slow':
-                    $slow_min_ms = $argv[++$i];
+                    if (is_numeric($argv[$i+1] ?? null)) {
```

Comment:
We're clearly rather lazy about it for many options, but I think the right fix here would be to validate the argument and report an error, rather than silently ignoring it. E.g. what `--context` does above.

Reply:
ok, rewritten same way as other, making n mandatory

(btw, I still think using [n] in help message is confusing, the common practice for [] means it is optional)

Reply:
I don't mind the help text being changed as well, what's the right syntax for this? `<n>` instead of `[n]`? Or just `n`?

Reply:
both <n> and N are commonly used, seems <n> better for us as we already use <file>, <workers> or <php>